<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QHBoxLayout" name="mainHorizontalLayout">
    <item>
     <spacer name="mainLeftSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="functionHorizontalLayout">
        <item>
         <spacer name="functionLeftSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="functionVerticalLayout">
          <item>
           <widget class="QLabel" name="functionLabel">
            <property name="text">
             <string>Функція:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="function1RadioButton">
            <property name="text">
             <string>3sin(5x) - 2cos(3x) + 3</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="function2RadioButton">
            <property name="text">
             <string>5^(34+x^5) - 5^(1-x^2)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="functionRightSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="argumentsLayout">
        <item>
         <widget class="QLabel" name="startLabel">
          <property name="text">
           <string>Початок</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="startSpinBox">
          <property name="decimals">
           <number>10</number>
          </property>
          <property name="minimum">
           <double>-1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="endLabel">
          <property name="text">
           <string>Кінець</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="endSpinBox">
          <property name="decimals">
           <number>10</number>
          </property>
          <property name="minimum">
           <double>-1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pointCountLabel">
          <property name="text">
           <string>Кількість точок</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pointCountSpinBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="calculationButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Обчислити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="resultTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="mainRightSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calculationButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>releaseCalculationButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>releaseCalculationButton()</slot>
 </slots>
</ui>
